TYPO3.Flow/Configuration/Settings.yaml

```diff
@@ -215,7 +215,7 @@ TYPO3:
       # Options for the default Doctrine persistence
       backendOptions:
         driver: 'pdo_mysql'
-        host: '127.0.0.1'
+        host: ''
```

Comment:
a NULL value would be more suiting than an empty string here, it clearly indicates an unset value unlike a empty string